### PR TITLE
{bio}[foss/2018b] nanofilt v2.5.0 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by fizwit** [comment]
I can not tell why the Travis build failed.

File: easybuild/easyconfigs/n/nanofilt/nanofilt-2.5.0-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+checksums = ['33c35aad2950145ef66bcf338e68c7c84a20ae62716db24f9c9ccbd881c9a277']
+
+exts_defaultclass = 'PythonPackage'
+dependencies = [
+    ('Python', '3.6.6'),
+    ('Biopython', '1.72', versionsuffix),
+]
+
+use_pip = True
+
```
Comment (vanzod) [inline easybuild/easyconfigs/n/nanofilt/nanofilt-2.5.0-foss-2018b-Python-3.6.6.eb]: @fizwit When building with the `PythonPackage` easyblock it is mandatory to add `download_dep_fail = True`. This is why Travis is complaining.

Comment (fizwit) [reply easybuild/easyconfigs/n/nanofilt/nanofilt-2.5.0-foss-2018b-Python-3.6.6.eb]: thanks!

File: easybuild/easyconfigs/n/nanofilt/nanofilt-2.5.0-foss-2018b-Python-3.6.6.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+homepage = 'https://github.com/wdecoster/nanofilt'
+description = """Filtering and trimming of long read sequencing data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/wdecoster/nanofilt/archive/v%(version)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['33c35aad2950145ef66bcf338e68c7c84a20ae62716db24f9c9ccbd881c9a277']
+
+exts_defaultclass = 'PythonPackage'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/n/nanofilt/nanofilt-2.5.0-foss-2018b-Python-3.6.6.eb]: this easyconfig has no extensions, please remove this (I suppose it's also why Travis is complaining about `exts_default_options`)

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/ab4ab3f757a954515bb0ff503d4c9208 for a full test report.

**Comment by migueldiascosta** [comment]
closing and reopening to trigger travis (it's been a while, new tests have been added)

**Comment by migueldiascosta** [comment]
Going in, thanks @fizwit!

**Pull request merged**